content/chapters/data/lab/quiz/memory-regions-vars.md

```diff
@@ -0,0 +1,17 @@
+# Variables in memory regions
+
+## Question Text
+
+In what memory regions are the `a`, `b`, `c`, `k` varoables stored?
+
+## Question Answers
+
+- a - .data, b - .bss, c - .stack, k - .heap
++ a - .data, b - .bss, c - .stack, k - .stack
+- a - .data, b - .data, c - .stack, k - .heap
+- a - .bss, b - .bss, c - .stack, k - .heap
```

Comment:
```suggestion
- a - .data, b - .bss, c - .stack, k - .heap

+ a - .data, b - .bss, c - .stack, k - .stack

- a - .data, b - .data, c - .stack, k - .heap

- a - .bss, b - .bss, c - .stack, k - .heap
```
Linter loves this!

Reply:
Also mind the `super-linter` errors. They say you should enclose the names of the sections in backticks like so: `.data`.